tests/unit/jetpack-search.spec.js

```diff
@@ -0,0 +1,38 @@
+import { mount, createLocalVue } from '@vue/test-utils'
```

Comment:
C'est bien de faire un copié collé de Jetpacks mais change au moins le composant avec to Search.vue